api-tests/inventory/agents_test.go

```diff
@@ -554,21 +554,23 @@ func TestQanAgentExporter(t *testing.T) {
 		}, getAgentRes)
 
 		// Test change API.
-		changeQANMySQLPerfSchemaAgentOK, err := client.Default.AgentsService.ChangeQANMySQLPerfSchemaAgent(
-			&agents.ChangeQANMySQLPerfSchemaAgentParams{
-				Body: agents.ChangeQANMySQLPerfSchemaAgentBody{
-					AgentID: agentID,
-					Common: &agents.ChangeQANMySQLPerfSchemaAgentParamsBodyCommon{
-						Disable:            true,
-						RemoveCustomLabels: true,
+		changeQANMySQLPerfSchemaAgentOK, err := client.Default.AgentsService.ChangeAgent(
+			&agents.ChangeAgentParams{
+				Body: agents.ChangeAgentBody{
+					QANMysqlPerfschemaAgent: &agents.ChangeAgentParamsBodyQANMysqlPerfschemaAgent{
+						AgentID: agentID,
+						Common: &agents.ChangeAgentParamsBodyQANMysqlPerfschemaAgentCommon{
+							Disable:            true,
+							RemoveCustomLabels: true,
```

Comment:
can we use the same approach @artemgavrilov introduced to not have 2 fields for each enable/disable and set/remove operations

Reply:
Good point - resolved with [this commit](https://github.com/percona/pmm/pull/2777/commits/1555a22ddc04e375463ead9663f40178b804ef7a)